### ISSUE TITLE
CVE-2021-20288: update ceph clients to v15.2.11

### ISSUE THREAD

**Issue by gman0** [ceph/ceph-csi#1995]
The latest ceph-csi images v3.3.0 contain `ceph version 15.2.10 (27917a557cca91e4da407489bbaa64ad4352cc02) octopus (stable)`. Just yesterday there was a security announcement CVE-2021-20288 and it's advised that all Octopus users upgrade to v15.2.11.

https://ceph.io/releases/v15-2-11-octopus-released/

**Comment by Madhu-1** [comment]
the docker image is updated in docker hub

```
docker inspect ceph/ceph:v15 |grep CEPH_POINT_RELEASE
                "CEPH_POINT_RELEASE=-15.2.11",
                "CEPH_POINT_RELEASE": "-15.2.11",
                "CEPH_POINT_RELEASE=-15.2.11",
                "CEPH_POINT_RELEASE": "-15.2.11",
```

next cephcsi release 3.3.1 should use 15.2.11 as the base image

**Comment by nixpanic** [comment]
More details at https://docs.ceph.com/en/latest/security/CVE-2021-20288/

This does not seem critical for the components that Ceph-CSI deploys. Warnings can get logged on the Ceph cluster until the Ceph-CSI clients have been updated.

**Comment by Madhu-1** [comment]
Fixed in #2009
